tests/src/authorization-tests.ts

```diff
@@ -82,43 +93,56 @@ test('Single sig spending condition uncompressed', () => {
   expect(serializedSpendingCondition).toEqual(spendingConditionBytes);
 });
 
-test('Single sig wpkh spending condition compressed', () => {
-  const addressHashMode = AddressHashMode.SerializeP2WPKH;
-  const nonce = new BigNum(345);
+test('Multi sig spending condition uncompressed', () => {
```

Comment:
Was the original test `Single sig wpkh spending condition compressed` intentionally removed?

Reply:
It was. It turns out that WPKH wasn't implemented yet. I'm not actually sure why that test was passing originally, but after my refactor it began to fail